From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
Date: Sun, 13 Jan 2013 10:47:51 +0100
Subject: fix_format-security_issue

File source/gameengine/Ketsji/KX_KetsjiEngine.cpp
needs a little adjustment to let blender build fine
on Debian.
---
 source/gameengine/Ketsji/KX_KetsjiEngine.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/gameengine/Ketsji/KX_KetsjiEngine.cpp b/source/gameengine/Ketsji/KX_KetsjiEngine.cpp
index 87683f8..fe20756 100644
--- a/source/gameengine/Ketsji/KX_KetsjiEngine.cpp
+++ b/source/gameengine/Ketsji/KX_KetsjiEngine.cpp
@@ -1499,7 +1499,7 @@ void KX_KetsjiEngine::RenderDebugProperties()
 	{
 		for (int j = tc_first; j < tc_numCategories; j++)
 		{
-			debugtxt.Format(m_profileLabels[j]);
+			debugtxt.Format("%s", m_profileLabels[j]);
 			m_rendertools->RenderText2D(RAS_IRenderTools::RAS_TEXT_PADDED,
 			                            debugtxt.Ptr(),
 			                            xcoord + const_xindent,
